tests/acceptance/features/apiGraphUserGroup/getUser.feature

```diff
@@ -1437,160 +1438,236 @@ Feature: get users
+      """
+      {
+        "type": "object",
+        "required": [
+          "value"
+        ],
+        "properties": {
+          "value": {
+            "type": "array",
+            "minItems": 2,
+            "maxItems": 2,
+            "uniqueItems": true,
+            "items": {
+              "oneOf": [
+                {
+                  "type": "object",
+                  "required": [
+                    "displayName",
+                    "id",
+                    "userType"
+                  ],
+                  "properties": {
+                    "displayName": {
+                      "type": "string",
+                      "const": "Alice Hansen"
+                    },
+                    "id": {
+                      "type": "string",
+                      "pattern": "^%user_id_pattern%$"
+                    },
+                    "userType": {
+                      "type": "string",
+                      "const": "Member"
+                    }
                   }
-                }
-              },
-              {
-                "type": "object",
-                "required": [
-                  "displayName",
-                  "id",
-                  "userType"
-                ],
-                "properties": {
-                  "displayName": {
-                    "type": "string",
-                    "enum": ["Alice Murphy"]
-                  },
-                  "id": {
-                    "type": "string",
-                    "pattern": "^%user_id_pattern%$"
-                  },
-                  "userType": {
-                    "type": "string",
-                    "enum": ["Member"]
+                },
+                {
+                  "type": "object",
+                  "required": [
+                    "displayName",
+                    "id",
+                    "userType"
+                  ],
+                  "properties": {
+                    "displayName": {
+                      "type": "string",
+                      "const": "Alice Hansen"
+                    },
+                    "id": {
+                      "type": "string",
+                      "pattern": "^%user_id_pattern%$"
+                    },
+                    "userType": {
+                      "type": "string",
+                      "const": [
+                        "Admin"
+                      ]
+                    }
                   }
                 }
+              ]
+            }
+          }
+        }
+      }
+      """
+
+  @env-config
+  Scenario Outline: search other users when OCIS_SHOW_USER_EMAIL_IN_RESULTS config is enabled
+    Given the config "OCIS_SHOW_USER_EMAIL_IN_RESULTS" has been set to "true"
+    And the administrator has assigned the role "<user-role>" to user "Alice" using the Graph API
+    When user "Alice" searches for user "Brian" using Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
```

Comment:
this should already be covered in `searchUserIncludingEmail.feature`, right?

Reply:
yes, correct. I removed it